updated changes in navbar

MansiBhayade opened the pull request:
Fixed navbar at the top, linked navbar to all the sections and redesigned navbar

MansiBhayade committed:
updated index.html

Fixed navbar at the top, linked navbar to all the sections and redesigned navbar

MansiBhayade committed:
updated style.css

VighneshManjrekar reviewed (changes requested):
<img width="1108" alt="image" src="https://user-images.githubusercontent.com/69366251/197490166-9b57bd99-bb1f-40dd-afe6-2612e6938175.png">
Add padding to the navbar and make `home` section of full height (height of viewport)

VighneshManjrekar reviewed (changes requested):
Don't change the color of the navbar

MansiBhayade committed:
Merge branch 'CSI-CATT-DMCE:master' into patch-1

MansiBhayade committed:
updated changes

added padding, brought navbar to original colour and height of home  image is equal to viewport

MansiBhayade commented:
@VighneshManjrekar updated changes

VighneshManjrekar commented on index.html:
```diff
@@ -60,18 +60,24 @@
                             </li>
                             <li class="nav-item">
-                                <a class="fs-5 fw-bold nav-link" href="joinus.html">Join Us</a>
+                                <a class="fs-5 fw-bold nav-link" href="#joinus">Join Us</a>
```
link join us page

MansiBhayade replied:
@VighneshManjrekar It is linked

MansiBhayade replied:
linked join us page

VighneshManjrekar reviewed (changes requested):
Refactor navbar on every page

MansiBhayade committed:
linked join us

VighneshManjrekar reviewed (commented):
Appreciate your PR but the new design is not approved by the UI/UX lead hence closing it. Kindly add your valuable contribution to other issues
